sonic_psu/psu_base.py

```diff
@@ -47,3 +47,75 @@ def get_psu_presence(self, index):
         """
         return False
 
+    def get_model(self, idx):
```

Comment:
you may align the method defied in new platform API, if eventually you still want to go with new plugins.